sdk/src/impl/whirlpool-impl.ts

```diff
@@ -111,8 +112,8 @@ export class WhirlpoolImpl implements Whirlpool {
     tickUpper: number,
     liquidityInput: IncreaseLiquidityInput,
     sourceWallet?: Address,
-    positionWallet?: Address,
```

Comment:
This was an unused parameter. Currently, the parameter labeled funder in the interface was being dropped.

This will change the behavior for any client putting a non-default funder here, so we could consider putting a placeholder in this position. I don't think the current code won't behave the way the user is expecting though.

Reply:
To test this I added the SDK as a dependency to the liquidity terminal, generated a keypair to pass in when creating a position, and verified it matched the token mint on the block explorer.

Reply:
good catch.
I agree that unused parameters that do not match the interface definition should be removed as implementation bug.